static/css/external-site.less

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+      border-top: 1px solid @gray-50;
```

Comment:
We should port this back to the global content bar, rather than using height and bg-color